packages/playground/vue/CustomBlock.vue

```diff
@@ -3,19 +3,22 @@
   <p class="custom-block">{{ t('hello') }}</p>
 </template>
 
-<script>
+<script lang="ts">
 import { getCurrentInstance } from 'vue'
 
 function useI18n(locale = 'en') {
   const instance = getCurrentInstance()
-  const resources = instance.type.i18n || { en: {} }
+  const resources = (instance.type as any).i18n || { en: {} }
   function t(key) {
     const res = resources[locale] || {}
     return res[key]
   }
   return { t }
 }
 
+/*
+export default
+*/
```

Comment:
```suggestion
/*
 *export default
 */
```
or
```suggestion
// export default
```

Or if not test relevant, just remove it? :thinking:

Reply:
It is test case =.=